tests/arm64/Tests.lst

```diff
@@ -90787,3 +90787,11 @@ Expected=0
 MaxAllowedDurationSeconds=600
 Categories=EXPECTED_PASS
 HostStyle=0
+
+[GitHub_16377.cmd_11720]
+RelativePath=JIT\Regression\JitBlue\GitHub_16377\GitHub_16377.cmd
```

Comment:
Looks like this should be JIT\Regression\JitBlue\GitHub_16377\GitHub_16377\GitHub_16377.cmd sorry

Reply:
https://github.com/CarolEidt/coreclr/pull/2 to address it.